pkg/action/uninstall.go

```diff
@@ -61,12 +62,15 @@ func (u *Uninstall) Run(name string) (*release.UninstallReleaseResponse, error)
 	}
 
 	if u.DryRun {
-		// In the dry run case, just see if the release exists
 		r, err := u.cfg.releaseContent(name, 0)
-		if err != nil {
+		switch {
+		case u.IgnoreNotFound && errors.As(err, &driver.ErrReleaseNotFound):
+			fallthrough
+		case err == nil:
+			return &release.UninstallReleaseResponse{Release: r}, nil
```

Comment:
nit: For readability it helps to have the success case first, followed by the error cases.

```suggestion
		case err == nil:
			return &release.UninstallReleaseResponse{Release: r}, nil
		case u.IgnoreNotFound && errors.As(err, &driver.ErrReleaseNotFound):
			fallthrough
```